lib/capybara/node/actions.rb

```diff
@@ -150,6 +150,11 @@ def unselect(value, options={})
       # @param [String] locator       Which field to attach the file to
       # @param [String] path          The path of the file that will be attached, or an array of paths
       #
+      # @option options [Symbol] match     The matching strategy to use (:one, :first, :prefer_exact, :smart)
+      # @option options [Boolean] exact    Match the exact locator name or accept a partial match
```

Comment:
exact only affects searches for text content of elements, so in this case it would only have an effect on searching label element contents